src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/TagletWriter.java

```diff
@@ -184,10 +185,20 @@ protected TagletWriter(boolean isFirstSentence) {
     protected abstract Content snippetTagOutput(Element element, SnippetTree snippetTag, StyledText text,
                                                 String id, String lang);
 
+    /**
+     * Returns the output for one or more {@code @spec} tags.
+     *
+     * @param element  the element that owns the doc comment
+     * @param specTags the array of @spec tags.
```

Comment:
Naked tags that aren't meant to be interpreted by javadoc make me uncomfortable.

Reply:
`{@code @spec}` similar to line 200?